lib/runtimeSettings.js

```diff
@@ -104,9 +104,26 @@ function getSettingsFile (settings) {
             broker: { ...settings.broker }
         }
     }
+    let contextStorage = ''
```

Comment:
```suggestion
    let contextStorage = `contextStorage: {
        default: "memory",
        memory: { module: 'memory' }
    },`
```

@knolleary @hardillb 

In this PR, `memory` (default) and `persistent` context stores are added to settings.js IF the EE lic is set. However this raises the question, should we add `default: memory` context regardless?

Reply:
Doesn't that happen automatically? If none is set then default is memory

Reply:
ok to go